Symfony/CS/Resources/phar-stub.php

```diff
@@ -15,8 +15,8 @@
         fwrite(STDERR, "HHVM needs to be a minimum version of HHVM 3.9.0\n");
         exit(1);
     }
-} elseif (!defined('PHP_VERSION_ID') || PHP_VERSION_ID < 50306) {
-    fwrite(STDERR, "PHP needs to be a minimum version of PHP 5.3.6\n");
+} elseif (!defined('PHP_VERSION_ID') || PHP_VERSION_ID < 50306 || PHP_VERSION_ID >= 70200) {
```

Comment:
these checks are exactly what i suggested for the composer file since they were more concise, but you didn't want them there. that's really why i'm asking